.github/workflows/ci.yml

```diff
@@ -160,10 +160,12 @@ jobs:
             python-version: '3.11'
         task:
           - name: Check Newsfragment
-            nox: check_newsfragment
+            run: |
+              nox -e check_newsfragment
+              nox -e draft_newsfragment >> $GITHUB_STEP_SUMMARY
```

Comment:
This is to see the pending release note rendered 

https://github.com/twisted/towncrier/actions/runs/6787483933#summary-18450436014

I hope it can help new contributors understand how the content of newsfragment is used.

In the future, we should have MD... but this is a first step.